src/scenes/ProgressReports/StatusHistory/WorkflowEventList.tsx

```diff
@@ -19,6 +19,8 @@ export const WorkFlowEventList = ({
     return null;
   }
 
+  const reversedEvents = [...events].reverse();
```

Comment:
Since we are depending the order to determine the previous status, I'd rather do that logic first. That way the reverse/forwards logic doesn't change the data anymore. We could also pull the first status from the API generated list.
```suggestion
  const list = events
    .map((event, i) => ({
      ...event,
      prevStatus: events[i - 1]?.status ?? StatusList[0]!,
    }))
    .reverse();
```

Reply:
where is this `StatusList[0]` you speak of?  Do you mean `ProgressReportStatusLabels`? it is not an array but dot notation works.